GUI/GUISlider.cpp

```diff
@@ -511,37 +523,17 @@ GUIPanel *GUISlider::GetPanel(void)
 //////////////////////////////////////////////////////////////////////////////////////////
 // Description:     Calculates the knob position and size.
 
-void GUISlider::CalculateKnob(void)
-{
-    m_KnobPosition = 0;
-    m_KnobSize = 0;
-
-    // Knob image not loaded yet
-    if (!m_KnobImage)
-        return;
-
-    if (m_Orientation == Horizontal) {
-        // Horizontal
-        m_KnobSize = m_KnobImage->GetWidth();
-        
-        if (m_Maximum-m_Minimum > 0) {
-            float V = (float)(m_Value-m_Minimum) / (float)(m_Maximum-m_Minimum);
-            m_KnobPosition = (float)(m_Width-m_KnobSize)*V;
-        }
-
-    } else {
-        // Vertical
-        m_KnobSize = m_KnobImage->GetHeight();
-        
-        if (m_Maximum-m_Minimum > 0) {
-            float V = (float)(m_Value-m_Minimum) / (float)(m_Maximum-m_Minimum);
-            m_KnobPosition = (float)(m_Height-m_KnobSize)*V;
-        }
-    }
-
-    // Clamp the knob position again for the graphics
-    m_KnobPosition = MAX(m_KnobPosition, m_EndThickness);
-    m_KnobPosition = MIN(m_KnobPosition, (m_Orientation == Horizontal ? m_Width : m_Height) - m_KnobSize - m_EndThickness);
+void GUISlider::CalculateKnob(void) {
+	if (!m_KnobImage)
+		return;
+
+	if (m_Maximum > m_Minimum) {
+		const bool orientation = (m_Orientation == Horizontal);
```

Comment:
Can you rename this to `orientationHorizontal` or `horizontalOrientation` or `orientationIsHorizontal` or something?
That way it'll be clearer at a glance what this means.